mw-webapp/src/component/text/Text.tsx

```diff
@@ -0,0 +1,45 @@
+import {ReactElement} from "react";
+
+/**
+ * Params for {@link Text}
+ */
+interface TextProps {
+
+  /**
+   * Displayed values
+   */
+  value: string | number;
+
+  /**
+   * Displayed placeholder
+   */
+  placeholder: string;
+
+  /**
+   * Function to render value
+   */
+  renderValue: (value: string | number) => ReactElement;
+
+  /**
+   * Function to render placeholder
+   */
+  renderPlaceholder: (placeholder: string) => ReactElement;
+}
```

Comment:
Need to add additional  optional prop className, because span can be differently styled in the different places